src/foam/nanos/menu/Menu.js

```diff
@@ -326,30 +359,35 @@ foam.CLASS({
     })
   ],
 
+  properties: [
+    'parent'
+  ],
+
   methods: [
     function init() {
       this.SUPER();
       var content;
 
       this.addClass(this.myClass())
-          .start()
-          .addClass(this.myClass('container'))
-          .start()
-              .addClass(this.myClass('background'))
-              .on('click', this.close)
-          .end()
-          .start()
-              .call(function() { content = this; })
-              .addClass(this.myClass('inner'))
-          .end()
-      .end();
+        .start()
+          .addClass(this.myClass('background'))
+          .on('click', this.close)
+        .end()
+        .start()
+          .call(function() { content = this; })
+          .addClass(this.myClass('inner'))
+        .end();
 
       this.content = content;
     },
 
     function open() {
-      this.document.body.insertAdjacentHTML('beforeend', this.outerHTML);
-      this.load();
+      if ( this.parent ) {
+        this.parent.add(this);
+      } else {
+        this.document.body.insertAdjacentHTML('beforeend', this.outerHTML);
```

Comment:
Did you test if this case still works? Is it even possible actually? Do we still need it?